### PR TITLE
Inputs design fix issue #21

### PR DESCRIPTION
**Pull request by aviv1620**
issue #21

### CODE REVIEW

**Comment by NoamGaash** [comment]
Cool!
could you please resolve the conflicts?

File: package.json [hunk 1]
```diff
@@ -3,9 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@mui/material": "^5.14.7",
+    "@mui/x-date-pickers": "^6.12.1",
     "antd": "^4.22.5",
```
Comment (NoamGaash) [inline package.json:10]: can it be removed?
(just asking)

**Review by NoamGaash — COMMENTED** [review]
please resolve conflicts, and make sure everything still works :)
thanks!

File: package.json (hunk 1 above)
Comment (aviv1620) [reply package.json:10]: no.
you probably need run again `yarn install`.

**Comment by aviv1620** [comment]
resolve, test, everything still works.

File: package.json (hunk 1 above)
Comment (NoamGaash) [reply package.json:10]: I was refering `ant`

**Review by NoamGaash — APPROVED** [review]

**Pull request merged**